readme-vars.yml

```diff
@@ -87,4 +87,5 @@ app_setup_block: |
 
 # changelog
 changelogs:
+  - { date: "10.09.24:", desc: "Add aarch64 support." }
```

Comment:
Date is the wrong way around, DD.MM.YY :)

Reply:
done, used to the US date format :D

Reply:
Whatever change you made has weirdly reformatted the readme-vars

Reply:
Sorry for that, format on save feature wanted to do that, I've to turn if off for that particular file, should be fine now.e